scripts/ci/test-examples.sh

```diff
@@ -39,8 +39,19 @@ cd "${CELER_SOURCE_DIR}/example/minimal"
 build_local
 ./minimal
 
-# Run Geant4 app example(s)
+# Run Geant4 app examples
 if [ -z "${CELER_DISABLE_ACCEL_EXAMPLES}" ]; then
+  if [ "$(echo $CELER_GEANT4_VERSION | awk -F. '{print $1*100 + $2}')" \
+      -ge 1100 ]; then
+    # Run offload-template only on Geant4 v11
+    cd "${CELER_SOURCE_DIR}/example/offload-template"
+    build_local
+    export CELER_DISABLE_PARALLEL=1
```

Comment:
The CI doesn't use MPI so this shouldn't be needed. If you build with MPI then just export it in your terminal before running this.